googleapiclient/http.py

```diff
@@ -559,6 +559,8 @@ def next_chunk(self, num_retries=0):
         content_range = resp['content-range']
         length = content_range.rsplit('/', 1)[1]
         self._total_size = int(length)
+      else:
+        self._total_size = int(resp['content-length'])
```

Comment:
we should probably check that `content-length` is there (yes, it *should* be, but things happen), and either fall back to 

1. an exception, or
2. length of what we can read from the response.

the latter of course has potential issues if we don't want to read a large response.